backend/services/deployments/docs/management_api.yml

```diff
@@ -64,6 +65,10 @@ paths:
         Returns a filtered collection of deployments in the system,
         including active and historical. If both 'status' and 'query' are
         not specified, all devices are listed.
+
+        DEPRECATED: we deprecated the endpoint due to an issue with "search" query
+          behavior. Please use the v2 /deployments/deployments end-point instead.
+          In the new endpoint, we replaced search parameter with "id" and "name" parameters.
```

Comment:
either endpoint in both places or end-point. histogram of usage:
```bash
#grep -rniHE end.*point backend/services/*/docs/*yml | grep -i [^a-zA-Z0-9/_]endpoint |wc -l
29
#grep -rniHE end.*point backend/services/*/docs/*yml | grep -i [^a-zA-Z0-9/_]end-point |wc -l
12
```

Reply:
also: maybe "with the id and name parameters" would sound better?

Reply:
done